### ISSUE TITLE
Missing ecore pkg-config pc files on debian

### ISSUE THREAD

**Issue by justinfx** [fyne-io/bootstrap#4]
Ubuntu 16.04
go 1.11.0

The current apt packages in the bootstrap don't provide the pc files needed to satisfy the pkg-config calls when building fyne. This extra package provides the pc files:

```
apt install libecore-dev
```

Although even after installing and building the test program, I get symbol failures:

```
../go/src/github.com/fyne-io/fyne/desktop/window.go:235:33 info.keycode undefined 
(type *_Ctype_struct__Ecore_Event_Key has no field or method keycode)
```

So I am not sure if I am even picking up the right version for Fyne.

**Comment by ajwillia-ms** [comment]
Hi,

Thanks for reporting this.
Can you confirm which version of libecore has been installed?

I have a patch locally for a debian system that I was playing with but I was not completely satisfied so it never got pushed.

Also: is the golang package from buster or from stretch-backports?

**Comment by justinfx** [comment]
Xenial installed libecore-dev 1.8.6-2.5:
https://packages.ubuntu.com/xenial/libecore-dev

And we are using Go 1.11 built from source.

**Comment by reneherrero** [comment]
Similar issue here,

Had to `sudo apt install libeina-dev libevas-dev libecore-dev` on Debian Stretch to get the pkg-config errors to go away, and now I'm stuck here:
```
# github.com/fyne-io/fyne/desktop
../desktop/window.go:235:33: info.keycode undefined (type *_Ctype_struct__Ecore_Event_Key has no field or method keycode)
```

**Comment by ajwillia-ms** [comment]
It was looking at updating the code to work with older EFL versions but I have found that some core features are not available until their 1.19 release. For this reason I'm going to update the minimum requirements for various distributions. The result for Ubuntu is that cosmic (18.10) will be required. Sorry to pull it forward so far but without the vector graphics capabilities Fyne struggles to look great!
